treeherder/intermittents_commenter/commenter.py

```diff
@@ -452,15 +450,12 @@ def build_bug_map(self, bugs, option_collection_map, start_day, end_day):
             if test_name:
                 manifest = all_tests[test_name][0]
                 if manifest:
-                    """
-                    # Bug 1972307 - disabled due to no comments
-                    tasks_count = self.get_task_labels_and_count(manifest, start_day, end_day)
-                    job_name = bug["job__signature__job_type_name"]
+                    tasks_count = self.get_task_labels_and_count(manifest, days)
+                    job_name = bug["job__signature__job_type_name"].rsplit("-", 1)[0]
```

Comment:
this will fail in  the scenario where we don't end with a `-<int>`.  so that needs to be checked first, or job_name.startswith(task_name)